src/runtime/pay-flow.js

```diff
@@ -77,6 +77,7 @@ export class PayStartFlow {
       },
       'i': {
         'startTimeMs': Date.now(),
+        'googleTransactionId': this.analyticsService_.getTransactionId(),
```

Comment:
Please add a `TODO` to accept it back from payments.

Reply:
Done